sdk/storage/azure-storage-blob/azure/storage/blob/aio/_blob_service_client_async.py

```diff
@@ -58,7 +58,11 @@
     from .._shared.models import UserDelegationKey
 
 
-class BlobServiceClient(AsyncStorageAccountHostsMixin, StorageAccountHostsMixin, StorageEncryptionMixin):  # type: ignore [misc]  # pylint: disable=line-too-long
+class BlobServiceClient(  # type: ignore [misc]
+    AsyncStorageAccountHostsMixin,
+    StorageAccountHostsMixin,
+    StorageEncryptionMixin
+):
```

Comment:
Personally I think having this multi-line is 😞 but also unnecessarily using line-too-long disable isn't great either.. Curious to see what @jalauzon-msft thinks..

Reply:
But also- I guess you made this change because of `next-pylint`, so our hand is forced here to not use the disable?

Reply:
Yeah, this is a tough one :D Unfortunately for this version of `pylint`, the `pylint: disable=line-too-long` won't fly because it's a "usless supression". We can either disable "useless supression" (which I am really not for) or we could separate into multiple lines, which I think even improves readability because it's easy to see the inherited classes. I know we don't have an established pattern for this but I like to see if this is a good idea or there's a better approach out there.

Reply:
Update: you can't supress "useless supression" lol

Reply:
This looks fine to me :)